app/upgrades/v8/unpool_whitelist.go

```diff
@@ -0,0 +1,59 @@
+package v8
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	gammkeeper "github.com/osmosis-labs/osmosis/v7/x/gamm/keeper"
+	superfluidkeeper "github.com/osmosis-labs/osmosis/v7/x/superfluid/keeper"
+)
+
+const ustDenom = "ibc/BE1BB42D4BE3C30D50B68D7C41DB4DFCE9678E8EF8C539F6E6A9345048894FCC"
+
+// RegisterWhitelistedDirectUnbondPools registers pools that are allowed to unpool
+// https://www.mintscan.io/osmosis/proposals/226
+// osmosisd q gov proposal 226
+func RegisterWhitelistedDirectUnbondPools(ctx sdk.Context, superfluid *superfluidkeeper.Keeper, gamm *gammkeeper.Keeper) {
+	// These are the pools listed in the proposal. Proposal raw text for the listing of UST pools:
```

Comment:
I know it might be a little late for this since the proposal is already up, but wouldn't it be just (if not more) appropriate to target LUNA pools as well given LUNA's hyperinflationary nature when UST is below peg?

It almost feels like the people who would be hit the hardest are the LUNA LPs, but that might have to go into a separate proposal.

@ValarDragon @sunnya97

Reply:
Gut-feeling is yeah, we should put up a proposal following the pattern of https://www.mintscan.io/osmosis/proposals/226

But with $LUNA pools.

Reply:
If you want to create a proposal for LUNA pools you better act quickly
https://www.mintscan.io/osmosis/proposals/227 is up to remove all incentives from **all Terra pools**
But nothing about unpooling LUNA pools

Reply:
I think theres several different things with Luna situation though:
- Governance didn't vote for it, can't do it unless governance gets super majority approval asap
- Luna was a volAsset, what distinguishes from chihuahua crashing. UST is different because people believe it was a stablecoin, and it had a peg. (Not saying morally if its right or wrong, but that its a different category)
- Supermajority of the damage has happened here, the Luna pool was over 1000x down when Terra turned off IBC.

Reply:
Actually agree with @ValarDragon on this now, especially after hearing @alexanderbez's input on the impact LUNA pools have already had. Even if we wanted to save what's left in LUNA pools, it doesn't seem as though there is much left to save. Feel free to resolve this discussion @ValarDragon unless there's anything else you want to add